code/modules/mob/living/simple_mob/subtypes/humanoid/mercs/mercs.dm

```diff
@@ -258,9 +258,80 @@
 
 	corpse = /obj/effect/landmark/mobcorpse/syndicatecommando
 
+	loot_list = list(/obj/item/gun/projectile/automatic/c20r = 100)
+	base_attack_cooldown = 5 // Two attacks a second or so.
+	reload_max = 20
+
 /mob/living/simple_mob/humanoid/merc/ranged/space/Process_Spacemove(var/check_drift = 0)
 	return
 
+// suppressors are just assholes and are intended to be a piss poor experience for everyone on both sides
+
+/datum/ai_holder/simple_mob/merc/ranged/suppressor
+	respect_alpha = FALSE // he really just shoots you
+	vision_range = 10 // plutonia experience
+
+/mob/living/simple_mob/humanoid/merc/ranged/space/suppressor // adminspawn only, and also Probably Going To Kill The Unprepared
+	name = "syndicate suppressor"
+	desc = "Geeze, weren't shotgun ops bad enough? At least when you fade these jerks you get a flashbang to the face."
+	icon_state = "syndi-ranged-space-sup"
+	icon_living = "syndi-ranged-space-sup"
+	armor = list(melee = 80, bullet = 65, laser = 50, energy = 15, bomb = 80, bio = 100, rad = 100) // this is the merc rig's stats
```

Comment:
Be careful about this. Regular Mercs use blood-red combat suit values. Suggest adding a new corpse that drops this new armor.

Reply:
to be fair explo gets ap weaponry fairly easily now

Reply:
That's irrelevant to my point. 
Problem isn't the new armor itself, it's that the merc corpse spawned will spawn normal blood red hardsuits instead of the new armor. Minor inconsistency.

Reply:
counterpoint: the only thing suppressors drop on death is a flashbang, or in the case of the elite, a stingbang. they have no corpse and no loot list, because they are constructs of spite